doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -592,12 +605,15 @@ def generate_payload_entries(self, assembly_inspector: AssemblyInspector) -> Dic
 
             entries: Dict[str, PayloadEntry]  # Key of this dict is release payload tag name
             issues: List[AssemblyIssue]
-            entries, issues = PayloadGenerator.find_payload_entries(assembly_inspector, arch,
-                                                                    self.full_component_repo())
+            entries, issues = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PUBLIC))
+
+            # Exclude issues for now, for private repo
+            private_entries, _ = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PUBLIC))
```

Comment:
the repo_type should be private?

Reply:
You're right! Thanks for the catch! Fixed in https://github.com/openshift-eng/art-tools/pull/410/commits/2cfe5c0b8cd05446df56a2e4420c85b80d61d53f